src/main/java/uk/gov/hmcts/dm/controller/StoredDocumentDeleteController.java

```diff
@@ -100,6 +103,13 @@ public ResponseEntity<CaseDocumentsDeletionResults> deleteCaseDocuments(
                 invalidRequestException
             );
             return new ResponseEntity<>(null, BAD_REQUEST);
+
+        } catch (final Exception exception) {
```

Comment:
Could me please remove the 500 response code? As we have spent decent amount of time in the past to remove the lines which return 500 response code. Catching the Exception and returning 500 response code hides the actual cause and result in wrong stats in Dynatrace which is monitored and raises alert emails.